pkgs/development/tools/misc/ccache/default.nix

```diff
@@ -15,12 +15,6 @@ let ccache = stdenv.mkDerivation rec {
 
   outputs = [ "out" "man" ];
 
-  # non to be fail on filesystems with unconventional blocksizes (zfs on Hydra?)
-  patches = [
-    ./fix-debug-prefix-map-suite.patch
-    ./skip-fs-dependent-test.patch
-  ];
```

Comment:
Since there is no comment on these patches that indicates whether they are (planned to be) upstreamed, nor where they come from, nor does the message of the commit that [added](https://github.com/NixOS/nixpkgs/commit/c24c0d560d252036e351900861d1aed050341ea1) them talk about that:

* Please comment in your commit message why these 2 patches can be removed (e.g. "implemented in version A.B.B", or even better link to ticket or release notes).

Reply:
I dug out the upstream commits which made these patches obsolete and mention them in the commit message.